pkg/apis/openebs.io/v1alpha1/iscsi.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+// ISCSISpec embeds common iscsi properties
+//
+// NOTE: This is intended to be used as embedded struct
+type ISCSISpec struct {
+	IQNSpec
+	TargetPortalSpec
+	TargetIPSpec
+	TargetPortSpec
```

Comment:
Spec under spec doesn't seem to be right ....here child spec contains only 1 parameter.

Reply:
This is because the child spec is also being shared in other structs.